modules/core/src/test/java/org/apache/ignite/testsuites/IgniteBasicTestSuite.java

```diff
@@ -208,6 +209,7 @@
     ListeningTestLoggerTest.class,
 
     CacheLocalGetSerializationTest.class,
+    ConsistentIdImplicitlyExplicitlyTest.class
```

Comment:
Looks like whitespace is expected before this line.

Reply:
@alamar But Ignite code guide speak other.
https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-MethodArguments

Reply:
This is not really a method argument, anyway it is preferred when neighboring  lines do not conflict

Reply:
whitespace or new line? anyway no one need here, code correct.

Reply:
@zstan it is obvious from above code that there is logical grouping. However, ConsistentIdImplicitlyExplicitlyTest does not group logically with ConsistentIdImplicitlyExplicitlyTest. I think it should be either separated by extra newline or grouped logically with some other classes about system properties or consistent id.

Reply:
Added new empty line.